i18n/ind/docusaurus-plugin-content-docs/current/lisk-tools/oracles.md

```diff
@@ -1,42 +1,44 @@
 ---
-title: Oracles
-slug: /lisk-tools/oracles
-description: Documentation for various blockchain oracles for Lisk.
-keywords:
-  [
-    Oracles,
-    Oracle,
-    Lisk,
-    Lisk Mainnet,
-    Lisk Testnet,
-    Lisk network,
-    Redstone,
-    price feeds,
-    data feeds,
-  ]
+
+title: Oracle  
+slug: /lisk-tools/oracles  
+description: Dokumentasi untuk berbagai orakel blockchain untuk Lisk.  
+keywords:  
+  [  
+    "Oracle",  
+    "Oracle",  
+    "Lisk",  
+    "Lisk Mainnet",  
+    "Lisk Testnet",  
+    "jaringan Lisk",  
+    "Redstone",  
+    "umpan harga",  
+    "umpan data",  
+  ]  
+
 ---
 
-# Oracles
+# Orakel
 
-[Oracles](https://ethereum.org/en/developers/docs/oracles/) provide offchain data onchain.
-This allows code running on a blockchain to access a wide variety of information.
+[Orakel](https://ethereum.org/en/developers/docs/oracles/) menyediakan data offchain ke onchain.  
+Ini memungkinkan kode yang berjalan di blockchain untuk mengakses berbagai macam informasi.
 
-The following Oracles support the Lisk network already:
+Berikut ini adalah orakel yang sudah mendukung jaringan Lisk:
 
 ## RedStone
 
-[RedStone](https://redstone.finance/) offers flexible Data Feeds for Lending Markets, Perpetuals, Options, Stablecoins, Yield Aggregators, and other types of novel DeFi protocols. 
+[RedStone](https://redstone.finance/) menawarkan Umpan Data fleksibel untuk Pasar Peminjaman, Perpetuals, Opsi, Stablecoins, Penggabungan Hasil, dan jenis protokol DeFi baru lainnya.
 
-Builders can choose how they want to consume the data from the following three dedicated models:
+Pembuat dapat memilih bagaimana mereka ingin mengonsumsi data dari tiga model khusus berikut:
 
-*   [RedStone Pull](https://docs.redstone.finance/docs/get-started/models/redstone-pull) (pull oracle) - less than 10s update time, broad spectrum of feeds, best for most use cases. All [Price Feeds](https://app.redstone.finance/#/app/tokens) are available for Lisk.
-*   [RedStone Push](https://docs.redstone.finance/docs/get-started/models/redstone-push) (push oracle) - for protocols designed for the traditional oracle interface, customizable heartbeat, and deviation threshold.
-*   [RedStone X](https://docs.redstone.finance/docs/get-started/models/redstone-x) - specifically for Perps and Options, highest update frequency, and front-running protection.
```

Comment:
@TalhaMaliktz we may want to use the long form of "Perpetuals" wordingsinstead of "Perps" for the English translation. Note that I cannot use suggest change feature as this is deleted lines on this PR.